Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -102,14 +102,21 @@ class NativeHandlersFactory {
     }()
     
     static var navigatorConfig: NavigatorConfig {
+        var nativeIncidentsOptions: MapboxNavigationNative.IncidentsOptions?
+        if let incidentsOptions = NavigationSettings.shared.liveIncidentsOptions,
+           !incidentsOptions.graph.isEmpty,
+           !incidentsOptions.apiURL.absoluteString.isEmpty {
```

Comment:
Why do we filter all IncidentsOptions if `apiURL` is an empty string, it's a completely valid case to pass only `graph` parameter.

Reply:
Good catch, I've missed this one